suites/suite_cam4.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="cam4" version="1.0">
+  <!-- 
+      CAM4 PHYSICS SUITE
+      Deep convection         ZM
+      Shallow convection      Hack
+      Macrophysics            RK    (not implemented)
+      Microphysics            RK    (not implemented)
+      Radiation               CAMRT (not implemented)
```

Comment:
We will be using RRTMGP as the radiation package for all CAM suites, so I might update the comment here:

```suggestion
      Radiation               RRTMGP (not implemented)
```